core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -871,6 +872,9 @@ protected void doRun() throws Exception {
             if (LOG_STARTUP_PERFORMANCE)
                 LOGGER.info(String.format("Took %dms for complete Jenkins startup",
                         System.currentTimeMillis()-start));
+
+            // Install the core UI Themes
+            CoreThemeIntsaller.installCoreThemes();
```

Comment:
This should use [`@Initializer`](http://javadoc.jenkins-ci.org/hudson/init/Initializer.html) on `CoreThemeIntsaller.installCoreThemes`.